style.css

```diff
@@ -157,6 +306,29 @@ body {
   transition: .5s;
 }
 
+@media (max-width: 1366px) {
+  .main__button {
+    margin: 18px 0 0 0;
+    left: 164px;
+    font: 500 14px/16px Roboto, arial, helvetica, sans-serif;
+  }
+}
+
+@media (max-width: 1024px) {
+  .main__button {
+    margin: 20px 0 0 2px;
+    left: 1px;
+    font: 500 14px/16px Roboto, arial, helvetica, sans-serif;
+  }
+}
+
+@media (max-width: 768px) {
+  .main__button {
+    margin: -6px 1px 0 0;
+    font: 500 14px/16px Roboto,arial,helvetica,sans-serif;
```

Comment:
Codacy found an issue: [Expected single space after "," in a single-line list (value-list-comma-space-after)](https://app.codacy.com/gh/WebHeroSchool/HTML_CSS_2.0-Artem1998/pullRequest?prid=6362272)